EDA-report-covid-19.ipynb

```diff
@@ -0,0 +1,1679 @@
+       "      <td>60.0</td>\n",
+       "      <td>12753.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>CAN</td>\n",
+       "      <td>2020-03-17</td>\n",
+       "      <td>120.0</td>\n",
+       "      <td>6446.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>CAN</td>\n",
+       "      <td>2020-03-18</td>\n",
+       "      <td>145.0</td>\n",
+       "      <td>9370.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>225</th>\n",
+       "      <td>CAN</td>\n",
+       "      <td>2020-10-27</td>\n",
+       "      <td>4109.0</td>\n",
+       "      <td>80889.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>226</th>\n",
+       "      <td>CAN</td>\n",
+       "      <td>2020-10-28</td>\n",
+       "      <td>2674.0</td>\n",
+       "      <td>55672.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>227</th>\n",
+       "      <td>CAN</td>\n",
+       "      <td>2020-10-29</td>\n",
+       "      <td>2699.0</td>\n",
+       "      <td>63142.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>228</th>\n",
+       "      <td>CAN</td>\n",
+       "      <td>2020-10-30</td>\n",
+       "      <td>2956.0</td>\n",
+       "      <td>62843.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>229</th>\n",
+       "      <td>CAN</td>\n",
+       "      <td>2020-10-31</td>\n",
+       "      <td>3457.0</td>\n",
+       "      <td>65562.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>230 rows × 4 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    iso_code        date  new_cases  new_tests\n",
+       "0        CAN  2020-03-12       10.0     4162.0\n",
+       "1        CAN  2020-03-15       68.0     2568.0\n",
+       "2        CAN  2020-03-16       60.0    12753.0\n",
+       "3        CAN  2020-03-17      120.0     6446.0\n",
+       "4        CAN  2020-03-18      145.0     9370.0\n",
+       "..       ...         ...        ...        ...\n",
+       "225      CAN  2020-10-27     4109.0    80889.0\n",
+       "226      CAN  2020-10-28     2674.0    55672.0\n",
+       "227      CAN  2020-10-29     2699.0    63142.0\n",
+       "228      CAN  2020-10-30     2956.0    62843.0\n",
+       "229      CAN  2020-10-31     3457.0    65562.0\n",
+       "\n",
+       "[230 rows x 4 columns]"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "covid.query(\"iso_code == 'CAN' and  new_tests > 0 and date <= '2020-10-31'\").reset_index(drop=True)"
```

Comment:
Should we also include the upper bound for the date? 
Like: `covid.query("iso_code == 'CAN' and  new_tests > 0 and '2020-03-11' < date <= '2020-10-31'").reset_index(drop=True)`

We said that we want to include the first non NaN occuence on the new tests column, right?